pkg/controller/operators/catalog/operator.go

```diff
@@ -1196,6 +1257,8 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 
 	logger.Info("syncing")
 
+	o.gcInstallPlans(logger, plan.GetNamespace())
```

Comment:
I think we want to bail out if this InstallPlan isn't the latest generation -- or has been deleted -- right?

Reply:
That's already handled by the rest of the installplan loop